File: app/src/main/java/avatar/com/mymvptestone/java/mvp/model/MainModel.java
package avatar.com.mymvptestone.java.mvp.model;

import avatar.com.mymvptestone.java.mvp.base.Model;

/**
 * $desc$
 *
 * @author Frank
 * email xiong.fu@avatarmind.com
 * created 19-4-4 下午1:58.
 */
public interface MainModel extends Model {

    /**
     * 从网络获取数据
     * @return
     */
    String getDataFromNet();

    /**
     * 暂停网络请求
     */
    void stopRequest();
}

File: app/src/main/java/avatar/com/mymvptestone/kotlin/mvp/model/MainModel.kt
package avatar.com.mymvptestone.kotlin.mvp.model

import avatar.com.mymvptestone.kotlin.mvp.base.Model

/**
 * $desc$
 * @author Frank
 * email xiong.fu@avatarmind.com
 * created 19-4-4 下午4:41.
 */
interface MainModel : Model {

    fun getDataFromNet(): String
    fun stopRequest()
}